colossus-metrics/src/main/scala/colossus/metrics/collectors/Counter.scala

```diff
@@ -20,8 +20,9 @@ object Counter {
 
   def apply(address: MetricAddress) = CounterParams(address)
 
-  implicit object LocalCounterGenerator extends Generator[LocalLocality, Counter, CounterParams] {
-    def apply(params: CounterParams)(implicit collector: ActorRef) = new LocalCounter(params, collector)
+  implicit object LocalCounterGenerator extends Generator[Counter, CounterParams] {
```

Comment:
Same with this..seems like the name should match the other generators, and just be CounterGenerator